plutus-tx-plugin/src/Language/PlutusTx/Plugin.hs

```diff
@@ -297,21 +296,19 @@ compileCoreExpr (opts, famEnvs) locStr codeTy origE = do
                 `GHC.App` bsLitPir
 
 runCompiler
-    :: forall uni m . (uni ~ PLC.DefaultUni, MonadReader (CompileContext uni) m, MonadState CompileState m, MonadQuote m, MonadError (CompileError uni) m, MonadIO m)
+    :: forall uni fun m . (uni ~ PLC.DefaultUni, fun ~ PLC.DefaultFun, MonadReader (CompileContext uni fun) m, MonadState CompileState m, MonadQuote m, MonadError (CompileError uni fun) m, MonadIO m)
     => PluginOptions
     -> GHC.CoreExpr
-    -> m (PIRProgram uni, UPLCProgram uni)
+    -> m (PIRProgram uni fun, UPLCProgram uni fun)
 runCompiler opts expr = do
     -- trick here to take out the concrete plc.error
-    tcConfigConcrete <-
-        runExceptT (PLC.TypeCheckConfig <$> PLC.getStringBuiltinTypes PIR.noProvenance)
+    let tcConfigConcrete = PLC.getDefTypeCheckConfig PIR.noProvenance
 
     -- turn the concrete plc.error into our compileerror monad
     stringBuiltinTCConfig <- liftEither $ first (view (re PIR._PLCError)) tcConfigConcrete
```

Comment:
This is prism-lift/mapping is not needed anymore. You can replace with ` stringBuiltinTCConfig <-  PLC.getDefTypeCheckConfig PIR.noProvenance`

Reply:
Done.